[DONE] Cleaned up docker-files and pinned GeoAlchemy2.

RichardB9 opened the pull request:
* Removed the docker base and docker 2.18
* Set the docker to qgis3.4.5 LTR
* Removed some unnecessary packages from docker
* Made sure qgis-setting files can be created in the docker
* ~~Added the source code in the dockerfile.~~

Commit:
Cleaned up docker-files and pinned GeoAlchemy2.

* Removed the docker base and docker 2.18
* Set the docker to qgis3.4.5 LTR
* Removed some unnecessary packages from docker
* Made sure qgis-setting files can be created in the docker
* Added the source code in the dockerfile.

Commit:
Updated travis.

renierkramer commented on Docker/qgis3.4.5/Dockerfile:
```diff
@@ -0,0 +1,93 @@
+RUN pip3 install --no-binary=h5py h5py
+
+RUN groupadd -g $GROUP_ID $GROUP_ID
+RUN useradd -m -d /home/$USERNAME --shell /bin/bash --uid $USER_ID --gid $GROUP_ID $USERNAME
```
add '-l' for large user id / group id ? :))

RUN useradd -m -d -l /home/$USERNAME --shell /bin/bash --uid $USER_ID --gid $GROUP_ID $USERNAME

renierkramer commented on Docker/qgis3/Dockerfile:
```diff
@@ -1,60 +0,0 @@
-RUN pip3 install --no-binary=h5py h5py
-
-RUN groupadd -g $GROUP_ID $GROUP_ID
-RUN useradd -m -d /home/$USERNAME --shell /bin/bash --uid $USER_ID --gid $GROUP_ID $USERNAME
```
add also '-l'  ??

Commit:
Fixed SIGSEGV (hopefully).

Seems to be caused by including the source code of ThreeDiToolbox
in the docker image, likey due to permission rights not set
correctly. Now fixed by removing the source code and just use the
mounting the source code as a volume.

Commit:
Added -l option to useradd.

Commit:
group 4 animation layers (#344)

Commit:
[Done] Renier remove spatial index (#342)

* fixed model and statistics layers by del spatial idx

* add v2_pipe to layer_tree, add progress bar to load schematisation

* fix pep8

* remove wrong docstring

* del create boundary_1d_view statement

* add pump_point, lateral1d, bound1d view plus new qml during sqlite load

* fix pep8

* edit Dockerfile

Commit:
fixed rubberbands for 1d2d (#347)

Commit:
update CHANGES
